src/pull_module/cmd_exec.cpp

```diff
@@ -21,15 +21,25 @@
 #include <string>
 
 namespace ovms {
-std::string exec_cmd(const std::string& command) {
+std::string exec_cmd(const std::string& command, int* returnCode) {
```

Comment:
```suggestion
std::string exec_cmd(const std::string& command, int& returnCode) {
```
use reference whenever possible